database/database.go

```diff
@@ -334,6 +334,16 @@ func (db *Database) GetInstanceByID(instanceID string) (instance launcher.Instan
 	return db.getInstanceInfoFromQuery("SELECT * FROM instances WHERE instanceID = ?", instanceID)
 }
 
+// GetInstanceInfoByID returns instance ident and service aos version by instanceID.
+func (db *Database) GetInstanceInfoByID(instanceID string) (cloudprotocol.InstanceIdent, uint64, error) {
```

Comment:
maybe better to add a name to the return argument? for example uint64 it's not obvious what it means